### PR TITLE
fix(core): patch CVE-2020-15168

### PR DESCRIPTION
**Pull request by mikicho**
https://nvd.nist.gov/vuln/detail/CVE-2020-15168

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
patch [CVE-2020-15168](https://nvd.nist.gov/vuln/detail/CVE-2020-15168)

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information
Should  I update the `package-lock` in the root folder? how should I do that?  `npm i` overrides all of it.

### CODE REVIEW

File: packages/core/package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     }
   },
   "dependencies": {
+    "@nuxt/opencollective": "0.3.2",
     "@nuxtjs/opencollective": "0.2.2",
```
Comment (jmcdo29) [inline packages/core/package.json:31]: Should this be removed like it was in the root `package.json`?

**Review by jmcdo29 — CHANGES_REQUESTED** [review]

File: packages/core/package.json (hunk 1 above)
Comment (mikicho) [reply packages/core/package.json]: Yeah, I removed it, I forgot to remove the old package from all the npm stuff :D

**Review by jmcdo29 — APPROVED** [review]
LGTM

**Comment by jmcdo29** [comment]
I'll check what the failure is on circleCI, but otherwise the change looks fine

**Comment by mikicho** [comment]
@jmcdo29 
No need to update the package-lock in the root folder?

**Comment by jmcdo29** [comment]
> @jmcdo29
> No need to update the package-lock in the root folder?

Yes, you need to update it. `npm i` should fix that. Tha will also fix the CircleCI failure

**Comment by mikicho** [comment]
@jmcdo29 I pushed the package-lock after `npm i`. but look on the change look too many changes, also the `lockfileVersion` file changed from 2 to 1.

**Comment by jmcdo29** [comment]
Interesting. May need @kamilmysliwiec's opinion on this. Looks like it did fix the build step though

**Comment by kamilmysliwiec** [comment]
Is there any difference between `@nuxtjs/opencollective` and `@nuxt/opencollective`?

**Comment by mikicho** [comment]
@kamilmysliwiec this is the same package:
https://github.com/nuxt-contrib/opencollective/issues/62

**Comment by kamilmysliwiec** [comment]
So is there any reason to switch from `@nuxtjs` to `@nuxt` as part of this PR?

**Comment by mikicho** [comment]
@kamilmysliwiec `@nuxtjs` is deprecated. they didn't even publish the patch to `@nuxtjs` until I opened that issue.

**Comment by mikicho** [comment]
@kamilmysliwiec `npm i` sohuld just work? what npm version do we use?

**Comment by kamilmysliwiec** [comment]
This issue was already addressed by Dependabot https://github.com/nestjs/nest/pull/5746 Anyway, thanks for your contribution!

**Comment by mikicho** [comment]
@kamilmysliwiec great!

Just, FYI they renamed the package's name. so we may not get future updates

**Pull request closed without merging**